src/i18n/en-US/intake.ts

```diff
@@ -138,6 +138,38 @@ const intake = {
         ]
       }
     }
+  },
+  contactDetailsForm: {
+    intakeProcessDescription:
+      'The EASi System Intake process can guide you through all stages of your project, connecting you with the resources, people and services that you need. Please complete and submit this CMS IT Intake form to engage with the CMS IT Governance review process. This is the first step to receive a CMS IT LifeCycle ID. Upon submission, you will receive an email promptly from the IT_Governance mailbox, and an IT Governance Team member will reach out regarding next steps.',
+    heading: 'Contact details',
+    requesterComponent: 'Requester Component',
+    businessOwner: {
+      name: 'CMS Business Owner Name',
+      helpText:
+        'This person owns a line of business related to this request and will champion the request moving forward',
+      component: 'CMS Business Owner Component'
+    },
+    productManager: {
+      name: 'CMS Project/Product Manager, or lead',
+      helpText:
+        'This person may be contacted for follow ups and to understand the state of the contract',
+      component: 'CMS Product Manager Component'
+    },
+    isso: {
+      question:
```

Comment:
went back and forth with label vs question here (and in the collab section as well), do not mind changing it.

Reply:
```suggestion
      label:
```

If you're fielding ideas, this might work. No strong opinions.

Reply:
yeah like I said I was going back and forth on label vs question, the more I think about it the more I like label. I'll make this change here and in the collab section.